src/Invoice_GenUI/Invoice_GenUI/App.xaml.cs

```diff
@@ -26,8 +26,11 @@ public App()
             services.AddSingleton<AddLineItemViewModel>();
             services.AddSingleton<ShowClientsViewModel>();
             services.AddTransient<ClientDetailsViewModel>();
+            services.AddSingleton<ShowInvoicesViewModel>();
+            services.AddTransient<InvoiceDetailsViewModel>();
```

Comment:
The singletons viewmodels are being injected into a view model and passing the values I need to that view model. When I had them as a transient the value being passed would always be null since I'm guessing a new instance is of the view model is created every time its requested but with singletons only once instance is created, which I want.